backend/src/services/linear/api/resolvers.py

```
@@ -11000,6 +11442,292 @@ def resolve_issueSubscribe(obj, info, **kwargs):
+    # Query for the comment by id
+    comment = session.query(Comment).filter(Comment.id == id).first()
+
+    if not comment:
+        raise Exception(f"Entity not found: Comment (id: {id})")
+
+    return comment
+
+
+@query.field("comments")
+def resolve_comments(
+    obj,
```

Ruff (0.14.4)

Unused function argument: `obj`

(ARG001)